.ci/stack-validate-pipeline-variables.yml

```diff
@@ -0,0 +1,88 @@
+#
+# Repos
+#
+
+#. git_repository (required): git@github.com:my/repo_to_validate.git
+#+ Git repository URL containing the stack to validate.
+git_repository: git@github.com:cycloid-community-catalog/stack-prometheus.git
+
+#. git_branch (required): master
+#+ Branch of the stack to validate Git repository.
+git_branch: master
```

Comment:
Shouldn't it be `((env))` ?

Reply:
This file is the variables from https://github.com/cycloidio/cycloid-stacks/blob/stacks/stack-validate/pipeline/variables.sample.yml

we don't have  `((env))` variable, only `git_branch: ($ environment $)` already render by the console